addons/point_of_sale/static/tests/tours/TicketScreen.tour.js

```diff
@@ -248,3 +248,51 @@ registry.category("web_tour.tours").add("FiscalPositionTwoTaxIncluded", {
             ProductScreen.isShown(),
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("MultipleRefundsTour", {
```

Comment:
```suggestion
registry.category("web_tour.tours").add("test_multiple_refunds", {
```
Recent convention in Pos, we want the tour to have the same name as the test, helps when searching the code.